src/libs/actions/Policy.js

```diff
@@ -187,6 +187,36 @@ function isAdminOfFreePolicy(policies) {
         && policy.role === CONST.POLICY.ROLE.ADMIN);
 }
 
+/**
+* Check if the user has any active free policies (aka workspaces)
+*
+* @param {Array} policies
+* @returns {Boolean}
+*/
+function hasActiveFreePolicy(policies) {
+    const adminFreePolicies = _.filter(policies, policy => policy
+        && policy.type === CONST.POLICY.TYPE.FREE
+        && policy.role === CONST.POLICY.ROLE.ADMIN);
+
+    if (adminFreePolicies.length === 0) {
+        return false;
+    }
+
+    if (_.some(adminFreePolicies, policy => !policy.pendingAction)) {
+        return true;
+    }
+
+    if (_.some(adminFreePolicies, policy => policy.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD)) {
+        return true;
+    }
+
+    if (_.some(adminFreePolicies, policy => policy.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE)) {
+        return false;
+    }
+
+    return true;
```

Comment:
I'm just wondering in what case this final return will be reached? Perhaps the 4th and 5th return could be combined into:

```
return _.some(adminFreePolicies, policy => policy.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE)
```

Reply:
Thanks for explaining. Perhaps a comment on the 2nd to last or last return would read simpler in that case

Reply:
Yep, ideal 👍